### DISCUSSION TITLE
对于 .chm 格式的资料，应该怎么处理

### DISCUSSION THREAD

**Discussion by PirateBook1** [banned-historical-archives/banned-historical-archives.github.io#1469] [Q&A]
@banned-historical-archives 对于 .chm 格式的文件，应该怎么处理

**Comment by banned-historical-archives** [comment]
现在只能人工处理，或者转换成pdf再自动录入

**Comment by banned-historical-archives** [comment]
https://github.com/banned-historical-archives/banned-historical-archives.github.io/issues/3971
